File: BT Training kiến thức về xử lý dữ liệu/huannv_Databinding/app/src/main/java/com/example/base_project/detailFim/DetailFilmActivity.java
package com.example.base_project.detailFim;

import android.content.Intent;
import android.os.Bundle;

import com.example.base_project.base.BaseActivity;
import com.example.base_project.detailFim.view.DetailFilmFragment;

import static com.example.base_project.main.MainActivity.ID_FILM;

public class DetailFilmActivity extends BaseActivity {

    @Override
    protected void initAct() {
        String idFilm = "";
        Intent intent = getIntent();
        if (intent != null) {
            idFilm = intent.getStringExtra(ID_FILM);
        }
        Bundle bundle = new Bundle();
        bundle.putString(ID_FILM, idFilm);
        pushView(DetailFilmFragment.getInstance(bundle));
    }
}

File: Trainning_Kotlin/app/src/main/java/com/example/myapplication/detailFilm/DetailFilmActivity.kt
package com.example.myapplication.detailFilm

import android.content.Intent
import android.os.Bundle
import androidx.core.os.bundleOf
import com.example.myapplication.MainActivity.Companion.ID_FILM
import com.example.myapplication.base.BaseActivity

class DetailFilmActivity : BaseActivity() {

    override fun initAct() {
        var idFilm : String = ""
        var intent : Intent = getIntent()
        idFilm = intent.getStringExtra(ID_FILM)
        var bundle : Bundle = Bundle()
        bundle.putString(ID_FILM, idFilm)
//        pushView()
    }
}